src/v/raft/coordinated_recovery_throttle.cc

```diff
@@ -92,10 +92,13 @@ void coordinated_recovery_throttle::setup_metrics() {
         _public_metrics.add_group(
           prometheus_sanitize::metrics_name("raft:recovery"),
           {sm::make_gauge(
-            "partition_movement_available_bandwidth",
-            [this] { return _throttler.available(); },
+            "partition_movement_consumed_bandwidth",
```

Comment:
right, I was chatting with Michal about this, let me add a new metric instead of replacing the old - not so useful- one, we can probably deprecate the old one later.